pages/organization-security/concepts.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Organizations Security - Concepts
+description: This page explains all the concepts related to Organizations Security
+tags: authentication saml security 
+dates:
+  validation: 2025-12-18
+---
+
+## Alias
+
+Each [Organization](#organization) can have an alias set up by an Organization Manager. Once set-up, all members can log in using a dedicated URL for the Organization using the alias, under the format [alias].account.scaleway.com
+
+## API key
+
+An API key is a unique identifier, used to authenticate requests made to the [Scaleway API](https://www.scaleway.com/en/developers/api/). An API key consists of an access key and a secret key. The access key is like a unique ID or username, and is not a sensitive piece of information. The secret key is more sensitive as it is like a password to authenticate the access key.
+
+API keys can have a validity duration defined by its creator. The maximum validity duration can also be enforced by an IAM administrator.
+
+## Console session
+
+A console session is an active, authenticated user session that allows interaction with the [Scaleway console](/account/concepts/#console). Console sessions duration can be limited by an [IAM administrator](#iam-administator).
+
+## Grace period
+
+The grace period is the time an [IAM Member](#members) has to comply with the security requirements that are enforced in your Organization before their account is automatically locked. The accounts can be manually unlocked by an Owner or [IAM Manager](#iam-manager). Upon regaining access, the grace period resets, giving IAM Members another chance to meet security requirements.
+
+## IAM manager
+
+An IAM manager can be the Owner of the Organization, or any IAM member with permission sets enabling them to perform administrative actions in the Organization, such as managing members or enforcing Security Requierements.
+
+
+## Identity Provider
+
+An Identity Provider (IdP) is a service that authenticates users and provides identity information to Scaleway to enable secure access through [Single Sign-On (SSO)](#single-sign-on)
+
+
+## Multi-Factor Authentication (MFA)
+
+Multi-factor authentication (MFA) is a security method that requires users to verify their identity using two or more independent factors, such as something they know, have, or are, before logging into an [Organization](/organizations-and-projects/concepts/#organization).
```

Comment:
Maybe we should link to the existing concept in Account as well, which provides more information.